CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+1.7 - September 24th, 2018
```

Comment:
If there are any unreleased changes, the toplevel list of changes should not have any heading. Rationale: The future release date is unknown and the next version may also be different (e.g., 1.7 → 2.0 instead of 1.7 → 1.8). See https://github.com/tiliado/nuvolaruntime/issues/474#issuecomment-424640716